### ISSUE TITLE
Acronym properties may have changed. Rerun to get them(acro) synchronized.

### ISSUE THREAD

**Issue by davelbit** [cgnieder/acro#249]
I am writing a report and have noticed that the \printacronyms command no longer updates the acronym list with new entries. The warning message above suggests the cause of the problem, but even after restarting the computer, updating texlive to the latest version and compiling the document several times, there were no changes.

That's why I was wondering if that problem has already occurred before, and what steps I could take to resolve that problem?

**Comment by dbranford** [comment]
Are you sure there are no other errors or warnings in the log file?  The rerun warning is often benign and (if not vanishing on multiple compilations) more likely to be another symptom of whatever is preventing the list from updating.

**Comment by davelbit** [comment]
1. Usage of package `parskip' together(scrbook) with a KOMA-Script class is not recommended.
2. Usage of package `fancyhdr' together(scrbook) with a KOMA-Script class is not recommended.
3. Acronym properties may have changed. Rerun to get them(acro) synchronized.

So that's all I get, but I suppose that it shouldn't have to do anything with the KOMA-Script conflict.

I could also add the whole log, but I don't know if that will be very helpful.

**Comment by dbranford** [comment]
Yeah, warnings look fine.  I see you're using `subfiles`, how are you loading the `./setup/packages.tex` and `./setup/acronyms.tex` files?  Is the `\printacronyms` call in `thesis.tex` or a different file?  Presumably all `\ac` uses occur in a `\subfile`-loaded file?

**Comment by davelbit** [comment]
The following contains the structure of the thesis.tex. All \ac uses occur in the subfiles of mainmatter

```
\input{setup/packages.tex}
\input{setup/acronyms.tex}

\begin{document}
\frontmatter
	\input{setup/titlepage.tex}
        \input{setup/declaration.tex}
        \input{setup/dedication.tex}
        \input{setup/acknowledgments.tex}
        \subfile{Abstract.tex}

        \tableofcontents
        \listoffigures
        \listoftables
	\printacronyms[name = {List of Acronyms}]
        \newpage

\mainmatter
        \subfile{Introduction}
        \subfile{Related Work}
        \subfile{Foundations}
        \subfile{Methods}
        \subfile{Results}
        \subfile{Discussion}
        \subfile{Conclusion}

\backmatter
        \subfile{Appendix}
        \FloatBarrier

        \begin{singlespace}
        	\sloppy
       		\printbibliography[heading=bibintoc,title={References}]
        \end{singlespace}
\end{document}
```

**Comment by dbranford** [comment]
Hmm, everything looks to be behaving, I wondered whether something might be generated in one `.aux` file and the list reading from another, but with this configuration everything looks ok.

```
\begin{filecontents}{setup/acronyms.tex}
\DeclareAcronym{foo}{short=foo,long=foo,}
\DeclareAcronym{bar}{short=bar,long=bar,}
\end{filecontents}

\begin{filecontents}{Text.tex}
\documentclass[acro_iss249]{subfiles}
\begin{document}
Text \ac{foo}
\end{document}
\end{filecontents}


\documentclass{scrbook}
\usepackage{acro}
\usepackage{subfiles}

\input{setup/acronyms.tex}

\begin{document}
\printacronyms
\subfile{Text}
\end{document}
```

Presumably `thesis.aux` is successfully being written to on compilation of `thesis.tex`? Not otherwise sure what might be causing issues. Have you tried compiling with acro 3.8 (https://github.com/cgnieder/acro/blob/9a56e414b0eb82ff89241decc66da9620f2292df/code/acro.sty)? Can you share what you get for `grep \ACRO thesis.aux`?
